src/lib/gadgets/gadgets.ts

```diff
@@ -33,4 +34,32 @@ const Gadgets = {
   rangeCheck64(x: Field) {
     return rangeCheck64(x);
   },
+
+  /**
+   * A (left and right) rotation is similar to the shift operation, `<<` and `>>` in JavaScript, just that bits are being appended to the other side.
+   * `direction` is a string which accepts either `'left'` or `'right'`, defining the direction of the rotation.
```

Comment:
Maybe clarify what left and right mean exactly in terms of endianness.

Reply:
Good call! Fixed in https://github.com/o1-labs/o1js/pull/1182/commits/fbe8aa59fb2c41e1713924298a5c296b68249a93